apps/web/scripts/docker/read-secrets.sh

```
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+set -euo pipefail
+if [ -f "/run/secrets/database_url" ]; then
+  export DATABASE_URL=$(cat /run/secrets/database_url)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)